mlx/directives/item_matrix_directive.py

```diff
@@ -57,14 +57,16 @@ def perform_replacement(self, app, collection):
         # External relationships are treated a bit special in item-matrices:
         # - External references are only shown if explicitly requested in the "type" configuration
         # - No target filtering is done on external references
-
-        relationships = self['type']
-        if not relationships:
+        mapping_via_intermediate = {}
+        if not self['type']:
             # if no explicit relationships were given, we consider all of them (except for external ones)
             relationships = [rel for rel in collection.iter_relations() if not self.is_relation_external(rel)]
-            external_relationships = []
+            ext_relationships = []
```

Comment:
I think external relationship name we had before is more descriptive...

Reply:
I agree. Done in cb2a31a9e4beb210e9245dec414b11be1ce2f334.